content/ja/handbook/engineering/devops/create/remote-development/index.md

```
@@ -0,0 +1,585 @@
+1. **次のステップ - イテレーション計画ミーティング：**
+   - リファインメント後、エピックはエピック内のすべての Issue が重み付けされる **Iteration Planning Meeting** に入る。
+   - このステージ後、エピックは完全に重み付けされ実行の準備ができていることを示すために<span style="color:#4dd787">ミント</span>でマークされる。
+
+#### 4. Iteration Planning Meeting
+
+**Iteration Planning Meeting** は、エピックボード上で<span style="color:#f3ad5d">アプリコット</span>としてマークされたエピック内の Issue をレビューして重み付けするチームの共同セッションです。これは [XP の "Weekly Cycle"](https://www.amazon.com/Extreme-Programming-Explained-Embrace-Change/dp/0321278658) または [Scrum の "Sprint Planning"](https://www.scrum.org/resources/what-is-sprint-planning) に類似しています。このプロセスにより、各リファインされたエピックが完全に理解され、スコープに収まり、チームの目標に沿っていることを確認できます。
+
+**ミーティングの目標：**
+
+- **Issue のレビューと重み付け：**
+  - 各 Issue について、ファシリテーターが説明を読み、チームが**簡潔に** Issue を議論して不明点を明確にします。ブロッキングする懸念/リスクが提起されなければ、チームはじゃんけんフィボナッチスケールで Issue を集合的に見積もり、集合的に合意したウェイトが割り当てられます。ウェイトの詳細については[使用するウェイト](#-what-weights-to-use)を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)